reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-70.md

```diff
@@ -0,0 +1,658 @@
+
+A null conditional operator permits member access, `?.`, or element access, `?[]`, to its operand
+only if that operand evaluates to non-null; otherwise, it returns null.
+
+[!NOTE] Since PowerShell allows `?` to be part of the variable name, formal specification of the
+variable name is required for using these operators. So it is required to use `{}` around the
+variable names like `${a}` or when `?` is part of the variable name `${a?}`.
+
+In the following example, the value of the member property **Status** is returned:
+
+```powershell
+$Service = Get-Service -Name 'bits'
+${Service}?.status
+Stopped
+```
+
+The following example will return null, without trying to access the member name **Status**:
+
+```powershell
+$service = $Null
+${Service}?.status
+```
+
+Similarly, using `?[]`, the value of the element will be returned:
+
+```powershell
+$a = 1..10
+${a}?[0]
+1
+```
+
+And when the operand is null, the element isn't accessed and null is returned:
+
+```powershell
+$a = $null
+${a}?[0]
+```
+
+For more information
+[About_Operators](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-7).
+
+## New view ConciseView and cmdlet Get-Error
+
+The display of error messages has been improved to enhance the readability of interactive and script
+errors with a new default view **ConciseView**. The views are user-selectable through the preference
+variable **$ErrorView**.
+
+With **ConciseView**, if an error is not from a script or parser error, then
+it's a single line error message:
+
+```powershell
+Get-Childitem -Path c:\NotReal
+```
+
+```Output
+Get-ChildItem: Cannot find path ‘C:\NotReal’ because it does not exist
+```
+
+If the error occurs during script execution or is a parsing error, a multiline error message will
+return that contains the error, a pointer and error message showing where the error is in that line.
+If the terminal doesn't support ANSI color escape sequences (VT100), then colors will not be
+displayed.
+
+![Error display from a script](./media/What-s-New-in-PowerShell-70/myscript-error.png)
+
+The default view in PowerShell 7 is **ConciseView**. The previous default view **NormalView** are
+user selectable by setting the preference variable **$ErrorView**.
+
+```powershell
+$ErrorView = 'NormalView' # Sets the error view to NormalView
+$ErrorView = 'ConciseView' # Sets the error view to ConciseView
+```
+
+[!NOTE] A new property **ErrorAccentColor** is added to **$Host.PrivateData** to support changing
+the accent color of the error message.
+
+A new cmdlet `Get-Error` provides complete detailed view of the fully qualified error when desired.
+By default the cmdlet displays the full details, including inner exceptions, of the last error that
+occurred.
+
+![Display from Get-Error](./media/What-s-New-in-PowerShell-70/myscript-geterror.png)
+
+The `Get-Error` cmdlet supports pipeline passed errors by using the built-in variable **$error**.
```

Comment:
What is a "pipeline passed error"?
Use backticks and proper case for `$Error`.
Change "by using" to "using"
Remove "will"